src/main/java/frc/robot/constants/JsonHandler.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.constants;
+
+import com.google.gson.Gson;
+import edu.wpi.first.wpilibj.Filesystem;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+//Static class, JsonHandler, uses gson to read and write the constants.json file
+public class JsonHandler {
+    private static String path = Filesystem.getDeployDirectory() + "/constants.json";
```

Comment:
check if deploy code overrides this folder

Reply:
wdym?